openff/recharge/esp/storage/_storage.py

```diff
@@ -28,13 +27,11 @@
     DBSoftwareProvenance,
 )
 from openff.recharge.esp.storage.exceptions import IncompatibleDBVersion
-
-if TYPE_CHECKING:
-    from openff.toolkit import Molecule
-
-    Array = numpy.ndarray
-else:
-    from openff.recharge.utilities.pydantic import Array, wrapped_array_validator
+from openff.recharge._annotations import (
+    ESP,
+    ElectricField,
+    Conformer as Coordinates,
```

Comment:
(not blocking) If we only ever use `Conformer` here, and we immediately rename it to `Coordinates`, maybe we should change its name to `Coordinates` in `_annotations.py`.